core/nodejsActionBase/runner.js

```diff
@@ -21,6 +21,9 @@
  * This file (runner.js) must currently live in root directory for nodeJsAction.
  */
 var util = require('util');
+var child_process = require('child_process');
+var fs = require('fs');
+var path = require('path');
```

Comment:
It's a delicate balance then.. Assuming a warm js container, `require`ing at the top saves some latency when initializing a zip-based action. Assuming a cold container, it wastes some time for non-zip actions?